docs/index.md

```diff
@@ -0,0 +1,14 @@
+Usine is a deployement tool that tries to make as little as possible for you not
+to stay on your way.
```

Comment:
I think the formulation is confusing here, maybe @vinyll can make a suggestion, I'm a bit out of idea.